src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1073,6 +1073,34 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
         return sdk_no_wait(no_wait, client.begin_create_or_update, resource_group_name, deployment_name, deployment)
     LongRunningOperation(cmd.cli_ctx)(client.begin_create_or_update(resource_group_name, deployment_name, deployment))
 
+    # Guest Attestation Extension and enable System Assigned MSI by default
+    if security_type and security_type.lower() == 'trustedlaunch' and enable_vtpm and\
+            enable_secure_boot and not disable_integrity_monitoring:
+        vm = get_vm(cmd, resource_group_name, vm_name, 'instanceView')
+        client = _compute_client_factory(cmd.cli_ctx)
+        if vm.storage_profile.os_disk.os_type == 'Linux':
+            publisher = 'Microsoft.Azure.Security.LinuxAttestation'
+        if vm.storage_profile.os_disk.os_type == 'Windows':
+            publisher = 'Microsoft.Azure.Security.WindowsAttestation'
+        version = _normalize_extension_version(cmd.cli_ctx, publisher, 'GuestAttestation', None, vm.location)
+        VirtualMachineExtension = cmd.get_models('VirtualMachineExtension')
+        ext = VirtualMachineExtension(location=vm.location,
+                                      publisher=publisher,
+                                      type_properties_type='GuestAttestation',
+                                      protected_settings=None,
+                                      type_handler_version=version,
+                                      settings=None,
+                                      auto_upgrade_minor_version=True,
+                                      enable_automatic_upgrade=None)
+        guest_attestation_extension = LongRunningOperation(cmd.cli_ctx)(
+            client.virtual_machine_extensions.begin_create_or_update(resource_group_name,
+                                                                     vm_name,
+                                                                     'GuestAttestation',
+                                                                     ext))
+        if guest_attestation_extension.provisioning_state == 'Succeeded':
+            logger.info('Guest Attestation Extension has been successfully installed by default')
+        else:
+            logger.error('Failed to install Guest Attestation Extension by default')
```

Comment:
Are these logger.info() and logger.error() calls printed to the console for the user to see?

Reply:
Yes, `logger.info()` will only print message when the users passe in `--debug` or `--verbose` parameters, and `logger.error()` will print information in all cases

Reply:
The purpose of adding logs is to let users know these default behaviors and avoid confusion when users are unaware these default behaviors